lib/matplotlib/tests/test_widgets.py

```diff
@@ -44,6 +44,36 @@ def test_rectangle_selector():
     check_rectangle(rectprops=dict(fill=True))
 
 
+@pytest.mark.parametrize('drag_from_anywhere, new_center',
+                         [[True, (60, 75)],
+                          [False, (30, 20)]])
```

Comment:
This is because dragging with `drag_from_anywhere=False` creates a new rectangle, which has a center (30, 20). I'll add some more comments to the test to make it clearer what's going on.